Author: Josh Triplett <josh@joshtriplett.org>
Description:
 Add new magic for gstreamer binary registry files (Closes: #559117).

diff -Naurp file.orig/magic/Magdir/misctools file/magic/Magdir/misctools
--- file.orig/magic/Magdir/misctools	2009-10-02 14:14:27.000000000 +0200
+++ file/magic/Magdir/misctools	2009-12-05 17:28:02.000000000 +0100
@@ -10,6 +10,12 @@
 4	string	gtktalog		GNOME Catalogue (gtktalog)
 >13	string	>\0			version %s
 
+# Summary: GStreamer binary registry
+# Extension: .bin
+# Submitted by: Josh Triplett <josh@joshtriplett.org>
+0	belong	0xc0def00d		GStreamer binary registry
+>4	string	x			\b, version %s
+
 # Summary: Libtool library file
 # Extension: .la
 # Submitted by: Tomasz Trojanowski <tomek@uninet.com.pl>
